<launch>
    <node name="trajectory_controller" pkg="hector_moveit_actions" type="trajectory_controller" respawn="false" output="screen"/>
    <node name="hector_navigator" pkg="hector_moveit_navigation" type="hector_navigator" respawn="false" output="screen">
        <param name="/planning_plugin" value="ompl_interface/OMPLPlanner"/>
    </node>
</launch>